src/python/gudhi/cubical_complex.pyx

```diff
@@ -145,6 +146,32 @@ cdef class CubicalComplex:
             persistence_result = self.pcohptr.get_persistence(homology_coeff_field, min_persistence)
         return persistence_result
 
+    def cofaces_of_cubical_persistence_pairs(self, homology_coeff_field=11, min_persistence=0, persistence_dim_max = False):
```

Comment:
Currently, when we want persistence under a different format, the interface is that the user has to call `persistence()` (with homology_coeff_field, min_persistence, etc), and then they can call other functions like `persistence_intervals_in_dimension` to get it in the format they like. I am not saying we can't change that, just making sure everyone is aware of it.

Reply:
OK. I modified the code accordingly to be more consistent. Now the function is more similar to `persistence_pairs()`